test/extended/util/framework.go

```diff
@@ -275,6 +286,17 @@ func ExtendedTestPath() string {
 	return os.Getenv("EXTENDED_TEST_PATH")
 }
 
+//ArtifactDirPath returns the value of ARTIFACT_DIR environment variable
+func ArtifactDirPath() string {
+	return os.Getenv("ARTIFACT_DIR")
+}
```

Comment:
the answer is "no" ... those methods are used outside the scope of It(), and hence will cause ginkgo to panic.